warehouse/static/js/warehouse/utils/search-filter-toggle.js

```diff
@@ -0,0 +1,29 @@
+export default () => {
+  const togglePanelDisplay = (display, event) => {
+    event.preventDefault();
+    var elements = document.querySelectorAll(".-js-dark-overlay, .-js-filter-panel");
+    for (var el of elements) {
+      el.style.display = display;
```

Comment:
oops, I had omitted a few words that really changed the meaning. I meant "run it by the lead"